lk2nd/device/dts/msm8909/apq8009w-wtp.dts

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	model = "Qualcomm APQ8009W";
+	qcom,msm-id = <QCOM_ID_APQ8009 0>, <QCOM_ID_APQ8009W 0>;
```

Comment:
Looks a bit of weird when you have 2 or more.

Reply:
> Looks a bit of weird when you have 2 or more.

The device would reboot if the `model` didn't exist in that location. In fact, I could remove the `model = "Mobvoi Tic...` and it would work, but to make the project cleaner, I did model= twice, one for to be able to boot, and the other to match with the rest of lk2nd

Unless you are looking at `qcom,msm-id `? This came from the stock dtb.

Reply:
it was about two msm-ids I believel, do you really need to match non-w variant of the soc if this device is w variant?